site/public/js/forum.js

```diff
@@ -675,6 +675,9 @@ function showHistory(post_id) {
                 var visible_user_json = JSON.stringify(visible_username);
                 info_name = JSON.stringify(info_name);
                 var user_button_code = "<a style='margin-right:2px;display:inline-block; color:black;' onClick='changeName(this.parentNode, " + info_name + ", " + visible_user_json + ", false)' title='Show full user information'><i class='fas fa-eye' aria-hidden='true'></i></a>&nbsp;";
+                if(!author_user_id){
```

Comment:
Nit: this condition is unclear. Perhaps define in a variable that an empty `author_user_id` string means that a limited access grader is viewing the post history.